data/core/init.lua

```diff
@@ -1602,6 +1602,7 @@ end)
 function core.run()
   local idle_iterations = 0
   while true do
+    if core.crash then error("Crashing as requested") end
```

Comment:
Would be nice if `core.crash` was initialized to false on `core.init()` to more easily have an idea of the available variables to expect on the `core` table.